part3/identifying-high-leverage-activities.qmd

```diff
@@ -0,0 +1,284 @@
+
+Here, we outline some potential ways to get started with both top-down and bottom-up evaluation suite design in order to setup a project for success. 
+
+**Top-down approach** 
+
+Generate various customer personas and scenarios, have an LLM create sample queries 
+
+```
+    "billing": {
+        "contexts": [
+            QueryContext("billing", True, True, "concerned customer", "bill inquiry"),
+            QueryContext("billing", True, True, "budget-conscious customer", "payment planning"),
+            QueryContext("billing", True, False, "confused customer", "charge explanation"),
+            QueryContext("billing", True, True, "data-heavy user", "usage verification"),  # Enhanced
+            QueryContext("billing", True, True, "traveling customer", "roaming charges"),
+            QueryContext("billing", True, True, "usage-monitoring customer", "usage tracking"),
+            QueryContext("billing", True, True, "business customer", "usage reporting"),
+            QueryContext("billing", True, True, "family plan customer", "usage analysis"),
+        ],
+        "base_scenarios": [
+            "customer sees unexpected charges on their bill",
+            "customer wants to know when payment is due",
+            "customer needs breakdown of current month charges",
+            "customer is questioning data usage amounts", 
+            "customer wants payment history for tax purposes",
+            "customer needs to understand prorated charges",
+            "customer is asking about auto-pay status",
+            "customer wants to dispute a specific charge",
+            "customer needs usage details for expense reporting",
+            "customer is planning data usage for upcoming month",
+            "customer wants to know their data usage for a specific month",
+            "customer needs to check voice minutes used in last billing cycle", 
+            "customer is asking about SMS usage over a date range",
+            "customer wants to compare usage between different months",
+            "customer needs total usage breakdown for expense reporting",
+            "customer is checking if they're approaching data limits",
+            "customer wants to analyze usage patterns over time",
+            "customer needs usage details for a specific billing period",
+            "customer is tracking usage to optimize their plan",
+            "customer wants to know peak usage periods",
+            "customer needs usage data for tax deduction purposes",
+            "customer is monitoring family member usage on shared plan",
+        ]
+    },
+```    
+**Bottoms-up approach**
+
+Conduct error analysis based on sample traces, then align LLM judge outputs with SME feedback to create automated tests
```

Comment:
Can we provide more information about how the team should collect these sample traces? This also varies by use case